okhttp/src/jvmMain/kotlin/okhttp3/internal/connection/RealConnectionPool.kt

```diff
@@ -118,6 +127,7 @@ class RealConnectionPool(
     connection.assertThreadHoldsLock()
 
     connections.add(connection)
+//    connection.queueEvent { connectionListener.connectEnd(connection) }
```

Comment:
What to do here? I think we need a test to check symmetry before we can drop this comment

Reply:
OK, I can add, unless you plan to.

Reply:
Lets do follow-up. This PR is big enough!